WcaOnRails/app/assets/stylesheets/incidents.scss

```diff
@@ -0,0 +1,65 @@
+// Place all the styles related to the Incidents controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.incident-form {
+  .nested-fields {
+    .comments-input {
+      @extend .form-control;
+      display: inline-block;
+      vertical-align: middle;
+      width: 50%;
+      margin-right: 15px;
+      margin-left: 15px;
+    }
+    .competition-selector {
+      width: 30%;
+      vertical-align: middle;
+      display: inline-block;
+    }
+  }
+}
+
+.incident-tag,
+.competition-tag {
+  display: inline-block;
+  &:hover {
+    cursor: pointer;
+  }
+  a {
+    all: unset;
+  }
+}
+
+.incidents-log-table-container {
+  .table-responsive {
+    overflow-x: visible;
+    .search,
+    #custom-search-toolbar {
+      @extend .pull-left;
+      padding: 5px;
+      input {
+        display: inline-block;
+      }
+    }
+    .selectize-input > input {
+      min-width: 100px;
+    }
+    .tags-cell,
+    .comps-cell {
+      vertical-align: top;
```

Comment:
Nitpick, but the `vertical-align: top` lets the "tag" header look off on smaller screens. Look's like it's safe to remove it completely.

Reply:
Weird, I remembered some visual issue without it but obviously now there is none.
I'll just remove it ^^